test/validation/api/ipsec/ipsec.c

```diff
@@ -937,6 +937,73 @@ void ipsec_check_out_one(const ipsec_test_part *part, odp_ipsec_sa_t sa)
 		odp_packet_free(pkto[i]);
 }
 
+static void ipsec_pkt_proto_err_set(odp_packet_t pkt)
```

Comment:
It would be good to have a separate patch for moving the existing code around. Then the actual change is not mixed with noise and it easier to read the commit later. That said, I am not sure this code should be moved here (as opposed to staying in where it is or moving to a new .c file).

Reply:
> It would be good to have a separate patch for moving the existing code around. Then the actual change is not mixed with noise and it easier to read the commit later.

Ack.  
 
> I am not sure this code should be moved here (as opposed to staying in where it is or moving to a new .c file).

As we are discussing the code organization in a separate email thread, once decision is made, will plan to make the changes as per that decision.